### ISSUE TITLE
Add legends to the arrow chart and scatter plot

### ISSUE THREAD

**Issue by sarahfossheim** [StateOfJS/StateOfJS-2020#56]
The arrow chart and scatter plot use color coding that's not explained. A legend should be added. 

<img width="1097" alt="Screenshot 2021-11-25 at 23 39 49" src="https://user-images.githubusercontent.com/10811350/143505386-43c5e689-12da-40f6-8e58-e57e5a9df948.png">

For the arrow chart it's a range, and the value and meaning of the values should be explained. What does it mean for a technology to be blue vs white vs red.

<img width="1030" alt="Screenshot 2021-11-25 at 23 23 09" src="https://user-images.githubusercontent.com/10811350/143505411-3b1a812d-732f-410b-ae15-75914670c8bb.png">

The scatter plot uses the colors to indicate what type of tool it is (front-end frameworks, data layer, testing, etc), should add which color maps to which category.
